progress/scripts/linkProgressReact.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+BLUE='\033[0;34m'
+BBLUE='\033[1;34m' #BOLD
+PURPLE='\033[0;35m'
+RED='\033[0;31m'
+BRED='\033[1;31m' #BOLD
+GREEN='\033[0;32m'
+BGREEN='\033[1;32m' #BOLD
+GRAY='\033[1;30m'
+NC='\033[0m' # No Color
+
+echo -e "${BLUE}Building components...${NC}"
+cd react/icons
+yarn && yarn build
+cd ../demo && yarn
+cd ..
+
+echo -en "${BLUE}Creating new folder in node_modules...${NC}"
+if [ -f ./demo/node_modules/@pxblue/react-progress-icons/package.json ]; then rm -rf "./demo/node_modules/@pxblue/react-progress-icons"; fi
+mkdir -p "./demo/node_modules/@pxblue/react-progress-icons"
+echo -e "${GREEN}Done${NC}"
+
+echo -en "${BLUE}Copying build output into node_modules...${NC}";
+cp -r ./icons/dist ./demo/node_modules/@pxblue/react-progress-icons/
+echo -e "${GREEN}Done${NC}"
+
+echo -en "${BLUE}Copying package.json into node_modules...${NC}";
+cp -r ./icons/package.json ./demo/node_modules/@pxblue/react-progress-icons
+echo -e "${GREEN}Done${NC}"
```

Comment:
Should this step occur during the `yarn build` in react/icons/package.json?

Reply:
And cp the readme as well?

Reply:
This doesn't need to occur in the build step...the build just builds the artifacts...the linking links them into the demo. The readme also doesn't do anything here. The only required files to copy into the node_modules are the /dist bundle and the package.json.

Reply:
You're thinking of the Angular project which is structured a little differently (and published from a different folder) so it has a couple more files that have to get copied/moved during the build step.